website/docs/foundations/typography/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,107 @@
+
+- `font-family: var(--token-typography-display-500-font-family);`
+- `font-size:var(--token-typography-display-500-font-size);`
+- `line-height:var(--token-typography-display-500-line-height);`
+- `font-weight:var(--token-typography-font-weight-bold);`
+
+Rather than calling upon each of these tokens in code every time text is styled, the [Text Component](/components/text) was created to replicate how Figma styles group text-based properties.
+
+## Font families
+
+Helios uses the default system fonts available on each OS. These system fonts are stable and support internationalization, code, tabular data, data visualization, etc. Additionally, they improve performance by reducing the number of resources to be loaded, which is important for large applications like those at HashiCorp.
+
+While text styling varies slightly across operating systems, users will have a consistent and expected experience within each OS.
+
+### Sans-serif
+
+![sans-serif on mac](/assets/foundations/typography/sans-serif-mac.png =660x*)
+
+![sans-serif on windows](/assets/foundations/typography/sans-serif-windows.png =660x*)
+
+![sans-serif on linux](/assets/foundations/typography/sans-serif-linux.png =660x*)
+
+### Monospace
+
+![monospace on mac](/assets/foundations/typography/monospace-mac.png =660x*)
+
+![monospace on windows](/assets/foundations/typography/monospace-windows.png =660x*)
+
+![monospace on linux](/assets/foundations/typography/monospace-linux.png =660x*)
+
+## Font weights
+
+In addition to font size, font weight helps create different levels of emphasis throughout the UI. Font weights include:
+
+- 400 (regular)
+- 500 (medium)
+- 600 (semi bold)
+- 700 (bold)
+
+Within these ranges, bold is used only for display and some code text styles, while regular is associated with body text styles which may include code.
+
+## Text styles
+
+### Display
+
+Display styles should be used for headings or hierarchical emphasis (like remaining balances). Unless used for hierarchical emphasis, if the display text looks like a page heading, it should use an [HTML heading tag](https://www.w3.org/WAI/WCAG22/Understanding/section-headings.html).
+
+Display text styles are available in the following sizes and weights:
+
+![Display 500 at 30px font size, with bold font weight. Display 400 at 24px font size, with bold, semibold and medium font weights. Display 300 at 18px font size, with bold, semibold and medium font weight. Display 200 at 16px font size, with semibold font weight. Display 100 at 13px font size, with medium font weight.](/assets/foundations/typography/typography-display-options.png)
```

Comment:
This is another situation where I would encourage moving the text out of an image and into code. Alt text doesn't support rich text like lists, so this is very hard to parse.

The text in the image could literally just be items in an HTML list that we style to not have bullets and each list item to have the correct display styles. 

```html
<ul>
  <li>Display 500 (30px) - Bold</li>
  <li>Display 400 (24px) - Bold</li>
  <li>Display 400 (24px) - Semibold</li>
  <li>Display 400 (24px) - Medium</li>
  // etc.
</ul>
```

Reply:
This goes for the body and code sections as well.